source/features/highlight-own-prs.tsx

```diff
@@ -0,0 +1,18 @@
+import select from 'select-dom';
+import features from '../libs/features';
+import {getUsername} from '../libs/utils';
+
+function init() {
+	for (const openedByEl of select.all(`.opened-by a[href*="${CSS.escape(getUsername())}"]`)) {
+		openedByEl.style.fontWeight = 'bold';
+	}
+}
+
+features.add({
+	id: 'highlight-own-prs',
+	include: [
+		features.isPRList
```

Comment:
You mentioned we could also highlight our own issues, that’s a good idea. Maybe the name should then be `highlight-own-issues-and-prs`